WIP: FCCEUD-58 Update Contributing to Pantheon

stoobie opened the pull request:
https://projects.engineering.redhat.com/browse/FCCEUD-58
Contributing to Pantheon (https://github.com/redhataccess/pantheon) targets all contributors, but it currently includes some guidance that is clearly targeting FCCEUD contributors without including similar content for other contributors.

E.g. Prerequisites include the following line: You must be a member of the pantheon-doc-authors team in the redhataccess group in GitHub.

benradey commented on README.md:
```diff
@@ -42,45 +42,63 @@ Fork the repository so that you can create and work with branches independently
 
+#### Code review
+
+???
```
A contributor would open a PR and use github's "Reviewers" gear icon to tag @redhataccess/pantheon-developers - then a developer would come inspect the code, make comments and have a discussion with the submitter, before ultimately deciding to accept or reject the PR. If the PR is approved, then the submitter has typically been the one to actually merge, although if we were to start accepting 3rd party submissions, then they wouldn't necessarily have that access. But that's the high-level process.

stoobie commented:
@benradey Please review my updates.

benradey commented:
LGTM, thanks @stoobie !

dvagnero commented on README.md:
```diff
@@ -42,44 +42,61 @@ Fork the repository so that you can create and work with branches independently
 Both the technical review and peer review processes take place in pull requests in GitHub.
 
+#### Documentation review
 After creating and labeling a pull request as outlined above, the developers review the pull request and add comments regarding technical accuracy. Writers receive a notification that comments have been added via email, and when all comments have been addressed, the developers change the label from **awaiting tech review** to **tech review passed**.
```
```suggestion
After creating and labeling a pull request as outlined above, the developers review the pull request and add comments regarding technical accuracy. Writers receive a notification that comments have been added via email, and when all comments have been addressed, the developers change the label from **awaiting tech review** to **tech review complete**.
```

stoobie replied:
@dvagnero  Yes, you're right, I made the suggested change. Please give approval again.

dvagnero commented:
Hey @stoobie, I completed the peer review. I have two suggestions that aren't related to your updates but worth mentioning. First one is the label **tech review complete** and second one is related to the placeholder for a new branch. FCCEUD<ID#> would be literally FCCEUD123 but we want FCCEUD-123 so what about something like "<JIRA_PROJECT-ID>, for example FCCEUD-123" ? This is just a nitpick and I believe that everyone in Red Hat follows this rule so they will understand the  FCCEUD<ID#> format. I'll wait for your answer on these two.

stoobie commented:
> Hey @stoobie, I completed the peer review. I have two suggestions that aren't related to your updates but worth mentioning. First one is the label **tech review complete** and second one is related to the placeholder for a new branch. FCCEUD<ID#> would be literally FCCEUD123 but we want FCCEUD-123 so what about something like "<JIRA_PROJECT-ID>, for example FCCEUD-123" ? This is just a nitpick and I believe that everyone in Red Hat follows this rule so they will understand the FCCEUD<ID#> format. I'll wait for your answer on these two.

Thanks for you suggestions @dvagnero! 

- The labels are defined by the developers, so that's what we've got. I think it's fine, though. We've been working with it for quite awhile now.
- I updated the example for naming a branch and added a hyphen. This example is actually biased towards a documentation branch, but I think it's still clear enough for everyone.

dvagnero reviewed (dismissed):
Approved :)

stoobie commented:
@redhataccess/pantheon-developers This is ready to merge.
